docs/api/app.md

```diff
@@ -396,7 +396,7 @@ Overrides the current application's name.
 
 ### `app.getLocale()`
 
-Returns the current application locale.
+Returns the current application locale. Possible return values are documented [here](../faq/app-locale.md)
```

Comment:
I'm not sure if we should put this file under `faq/`, other docs like `chrome-command-line-switches.md` and `environment-variables.md` are put under `api/`.